pytorch_lightning/core/mixins/device_dtype_mixin.py

```diff
@@ -177,7 +179,9 @@ def __update_properties(
         self, device: Optional[torch.device] = None, dtype: Optional[Union[str, torch.dtype]] = None
     ) -> None:
         def apply_fn(module: Union["DeviceDtypeModuleMixin", Module]) -> None:
-            if not isinstance(module, DeviceDtypeModuleMixin):
+            # TODO: Find why `isinstance(module, DeviceDtypeModuleMixin)` doesn't
```

Comment:
is the changelog outdated?

Reply:
This is the `materialized model not being moved to the device`. Somehow, the isinstance(module, DeviceDtypeModuleMixin) doesn't work with the materialised model.